cura/CuraApplication.py

```diff
@@ -145,13 +146,15 @@ def __init__(self):
         Resources.addStorageType(self.ResourceTypes.UserInstanceContainer, "user")
         Resources.addStorageType(self.ResourceTypes.ExtruderStack, "extruders")
         Resources.addStorageType(self.ResourceTypes.MachineStack, "machine_instances")
+        Resources.addStorageType(self.ResourceTypes.DefinitionChangesContainer, "machine_settings")
```

Comment:
Just a minor comment, but I would prefer we use "definition_changes" for this. We should match the storage name to our internal naming as much as possible in my opinion.